recipe/meta.yaml

```
@@ -32,13 +32,16 @@ source:
 build:
   number: {{ build }}
   string: {{ build_string }}
+  skip: true  # [py<312]
```

```suggestion
  skip: true  # [ffmpeg == 6]
  skip: true  # [py<312]
```

I think this should skip ffmpeg6 for you, but you should be able to build with both.